lib/enumerize/activerecord.rb

```diff
@@ -59,5 +65,19 @@ def update_all(updates)
         super(updates)
       end
     end
+
+    class Type < ActiveRecord::Type::Value
```

Comment:
@tkawa Sorry, I'm not sure that I understand how to reproduce the bug you are describing. Can you please push a test application reproducing it?